index.js

```diff
@@ -0,0 +1,116 @@
+'use babel';
+
+import { CompositeDisposable } from 'atom';
+import { spawn } from 'child_process';
```

Comment:
I would suggest you do [`atom-linter::exec`](https://npmjs.org/package/atom-linter)